controller/usersController.ts

```diff
@@ -0,0 +1,65 @@
+import { Request, Response } from "express";
+import { StatusCodes } from "http-status-codes";
+import pool from "../postgresql";
+import { joinQuery, getQuery } from "../model/usersModel";
+import admin from "../firebaseAdmin";
```

Comment:
요런 느낌으루 한번 정리해보시는걸 추천드릴게요!
```suggestion
import { Request, Response } from "express";
import { StatusCodes } from "http-status-codes";

import { joinQuery, getQuery } from "../model/usersModel";

import pool from "../postgresql";
import admin from "../firebaseAdmin";
```